tree-sitter-verilog/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: tree-sitter-verilog
+  version: {{ GIT_FULL_HASH }}
+
+source:
+  git_url: https://github.com/tree-sitter/tree-sitter-verilog.git
+
+build:
+  # number: 201803050325
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20180305_0325
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - TRAVIS
+
+requirements:
+  build:
+    - gcc_linux-64
```

Comment:
These are the old way of specifying the compiler. You should be using `{{ c_compiler }}` now - see [the Yosys meta.yml file](https://github.com/SymbiFlow/conda-packages/blob/master/yosys/meta.yaml) for an example.

Reply:
done